content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,145 @@
----------------------------------------
+Click :icon:`fa-search` :guilabel:`Search Messages` to search messages in the conversation.
 
-To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
-`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
-on their communication settings.
+Clicking :icon:`fa-thumb-tack` :guilabel:`Pinned Messages` shows pinned messages, and clicking
+:icon:`fa-paperclip` :guilabel:`Show Attachments` shows attachments shared in the conversation.
 
-.. image:: discuss/chat-windows.png
-   :alt: View of a couple of chat window messages for Odoo Discuss.
+Click :icon:`fa-user-plus` :guilabel:`Add Users` to invite more people to the direct message
+conversation. Click :icon:`fa-cog` :guilabel:`Show Call Settings` to configure voice chat settings.
 
-.. tip::
-   When a user is mentioned, the search list (list of names) suggests values first based on the
-   task's followers, and secondly on employees. If the record being searched does not match with
-   either a follower or employee, the scope of the search becomes all partners.
+At the top of the direct message window, click the name of the direct message to change the group
+name, and choose to add a description in the adjacent :guilabel:`Add a description` field.
 
 User status
 -----------
 
-It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *status*. The status is shown on the left side of a contact's name on the
-:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
+It is helpful to see what colleagues are up to, and how quickly they can respond to messages, by
+checking their *status*. The status is shown on the right side of a contact's name on the sidebar,
+on the *messaging menu*, and when listed in the *Chatter*.
 
 - Green = online
 - Orange = away
 - White = offline
 - Airplane = out of the office
 
-.. image:: discuss/status.png
-   :height: 300
-   :alt: View of the contacts' status for Odoo Discuss.
+Delete a direct message conversation
+------------------------------------
+
+To delete direct message conversations, click the :guilabel:`X` next to the name in the
+:guilabel:`Direct Messages` section of the sidebar.
+
+.. note::
+  This does **not** delete the :guilabel:`Direct Messages` in the conversation. The direct message
+  conversation's history is visible when another direct message with the same person, or group, is
+  created.
+
+.. _discuss/meetings:
+
+Meetings
+========
+
+In **Discuss**, :guilabel:`Meetings` are video calls. To start a meeting from the **Discuss**
+application home page, click :guilabel:`Start a meeting` in the top-left corner, and select who to
+invite to the meeting, via the :guilabel:`Invite People` drop-down window that appears. To start a
+meeting from a direct message, click the :icon:`fa-phone` :guilabel:`Start a Call` icon in the
+top-right.
```

Comment:
```suggestion
top-right corner.
```

Reply:
do you think it would be beneficial to mention that the use of microphones and/or video camera on the device being used must be given access to Odoo before a meeting like this could start? That could be a very helpful admonition to include here.

Reply:
Also, it should be clear that once all desired parties have been selected from the Invite People drop-down menu that appears, that the 'Invite to Group Chat' button must be clicked in order to officially send that invite/remove that pop-up. Also, mention how a copy/paste link to the meeting can be found at the bottom of the pop-up as well.